<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_update</class>
 <widget class="QDialog" name="Dialog_update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Photo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>276</width>
     <height>368</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_ID">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_Password"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_Name"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Password">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Gedner">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label_Signature">
      <property name="text">
       <string>个性签名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_ID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Account">
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_Account"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_Gender">
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QTextEdit" name="textEdit_Signature"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_SetPhoto">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>设置头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ClearPhoto">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Check">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
